java/src/jmri/jmrit/beantable/oblock/TableFrames.java

```diff
@@ -435,9 +434,19 @@ protected JInternalFrame makeBlockFrame() {
             OBlockTableModel.DELETE_COL, OBlockTableModel.REPORT_CURRENTCOL,
             OBlockTableModel.PERMISSIONCOL, OBlockTableModel.UNITSCOL});
 
-        TableRowSorter<OBlockTableModel> sorter = new TableRowSorter<>();
```

Comment:
The problem is that TableRowSorter needs the model as parameter. Restore this line and change it to:
```
TableRowSorter<OBlockTableModel> sorter = new TableRowSorter<>(_oBlockModel);
```
The other TableRowSorter initializers need to be similarly fixed.